Content.Server/Salvage/SalvageSystem.Magnet.cs

```diff
@@ -14,7 +14,7 @@ namespace Content.Server.Salvage;
 public sealed partial class SalvageSystem
 {
     [ValidatePrototypeId<RadioChannelPrototype>]
-    private const string MagnetChannel = "Supply";
+    private const string MagnetChannel = "Engineering"; // ParkStation - Supply to Engineering for EngineeringSalvage
```

Comment:
This comment is unintelligible :P
```suggestion
    private const string MagnetChannel = "Engineering"; // ParkStation - Salvage is in Engineering
```